src/Symfony/Component/Mailer/Bridge/Sendgrid/Header/SuppressionGroupHeader.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mailer\Bridge\Sendgrid\Header;
+
+use Symfony\Component\Mailer\Exception\InvalidArgumentException;
+use Symfony\Component\Mime\Header\UnstructuredHeader;
+
+/**
+ * @author Kieran Cross
+ */
+final class SuppressionGroupHeader extends UnstructuredHeader
+{
+    public function __construct(
+        private int $group_id,
```

Comment:
```suggestion
        private int $groupId
```
No snake case please

Reply:
Sorry, hard habit to break! Back to camel case - 38bb821